roles/test_operator/defaults/main.yml

```diff
@@ -43,6 +43,34 @@ cifmw_test_operator_tempest_network_attachments: []
 cifmw_test_operator_tempest_tests_include_override_scenario: false
 cifmw_test_operator_tempest_tests_exclude_override_scenario: false
 cifmw_test_operator_tempest_workflow: []
+
+# Enabling SRBAC by default, in jobs where this does not make sense should be turned off explicitly
+cifmw_tempest_tempestconf_config_defaults:
+  deployerInput: |
+    [enforce_scope]
+    # this doesn't work for nova team, let's enable srbac everywhere else
+    # and revisit nova later
+    # nova = true
```

Comment:
if this is form my feedback you have not understood what i said at all.

i explicitly said that enforce_scope is just a minor part of SRBAC and that the primary change was the enforce_new_defaults option.

so that twe need to test both.

it was then pointed out by others that this si just unfortunate nameing in tempeast and that enforce_scope is being used to feature gate both features.

i repeatliy tried to make it clear that i was not objecting to enabling these test, and highlighting that test only scope enforcement would not be sufficient to validate srbac as it only a minor part of it. since this apparently it testing both the new default and scope enforcement unlike what the config option name implies i don't see why we would not have this enabled for nova as well.